.github/workflows/ci.yml

```diff
@@ -25,13 +25,30 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - python: 3.6
+          - framework: tensorflow
+            python: 3.6
             tensorflow: 1.15.3
             keras: 2.2.5
-          - python: 3.7
+          - framework: keras
+            python: 3.7
             tensorflow: 2.2.0
             keras: 2.3.1
-    name: Run Tests (Python ${{ matrix.python }})
+          - framework: kerastf
+            python: 3.7
+            tensorflow: 2.2.0
+            keras: 2.3.1
+          - framework: pytorch
+            python: 3.7
+          - framework: scikitlearn
+            python: 3.7
+          - framework: mxnet
+            python: 3.7
+          - framework: legacy
```

Comment:
This is perfect. The only thing is that we need an extra matrix element identical to the `legacy` one but instead having the framework value set to `tensorflow`. This will make sure that the new non legacy tests, are also ran using tensorflow2

Reply:
Just added an extra `tf_version` matrix argument to do just this.